CLI crashes after global install

The published CLI reads `../package.json` at runtime to get the version. That works in the repo, but it breaks after a global install because the built chunk no longer has that file next to it.

Repro:
- install websxa globally
- run `websxa`
- see `Cannot find module '../package.json'`

The fix should keep version lookup inside the build output instead of depending on a runtime package.json path.

--- a/src/version.ts
+++ b/src/version.ts
@@ -1,6 +1,3 @@
-import { createRequire } from 'node:module'
-
-const require = createRequire(import.meta.url)
-const pkg = require('../package.json') as { version: string }
+import pkg from '../package.json' with { type: 'json' }
 
 export const version: string = pkg.version

